index.md

```diff
@@ -24,6 +28,7 @@ footer: >   # Translate all the words below, including "Date:" and "Editor:". Do
 # end of translation instructions
 ---
 
+@@@@@@@  +  new icons for links-to-w3c-page-without-wai-nav @@@@@@@
```

Comment:
The new icon is used automatically when `{% include different.html %}` is used. (Can have label attribute, like {% include different.html label="Looks differently" %}).

Reply:
Please document this where it is easy for editors to find. Thanks!

Reply:
Happy to do so when writing documentation is a priority. It has explicitly not been one before.